### PR TITLE
using super(), simplifying math

### PR DESCRIPTION
**Pull request by kejohnso**

### CODE REVIEW

File: openquake/hazardlib/gsim/munson_thurber_1997.py [hunk 1]
```diff
@@ -106,13 +106,18 @@ class MunsonThurber1997Hawaii(MunsonThurber1997):
     Seismic-hazard maps for Hawaii. US Geological Survey; 2000.
     """
 
+    DEFINED_FOR_INTENSITY_MEASURE_TYPES = set([PGA, SA])
+
```
Comment (micheles) [inline openquake/hazardlib/gsim/munson_thurber_1997.py]: You can also use `{PGA, SA}`.

**Comment by klunk386** [comment]
LGTM

File: openquake/hazardlib/gsim/munson_thurber_1997.py (hunk 1 above)
Comment (kejohnso) [reply openquake/hazardlib/gsim/munson_thurber_1997.py]: Is this syntax better? In other gsims I have checked as examples, I only see ([...])

Comment (micheles) [reply openquake/hazardlib/gsim/munson_thurber_1997.py]: The old syntax is for Python 2, now we are using Python 3.

**Pull request merged**